lib/components/Stepper/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import clsx from "clsx";
+import "./stepper.scss";
+import { LabelPairedCheckMdFillIcon } from "@deriv/quill-icons";
+import { Text } from "@components/Typography";
+import { TBasicStepperProps } from "./types";
+import { VerticalStepper } from './vertical-stepper';
+import { HorizontalStepper } from './horizontal-stepper';
+import { TypographyProps } from "@components/Typography/base";
+
+const IconOk = ({ size }: { size: TypographyProps['size'] }) => (
+    <LabelPairedCheckMdFillIcon
+        fill="var(--semantic-color-slate-solid-surface-frame-low)"
+        width={size === "sm" ? 11 : size === "md" ? 13 : 14}
+    />
+);
+
+export const BasicStepper = ({
+    currentStep,
+    Icon = IconOk,
+    size = "md",
+    labels = [],
+    allSteps,
+    lineSize = "md",
+}: TBasicStepperProps) => {
+    const steps = Object.values(allSteps).filter(
+        (value) => typeof value === "number"
+    );
+
+    return (
+        <div className="steps-container" >
```

Comment:
Can we have the main container have a `quill-` prefix.

Reply:
thank you🙇
improved✅